climakitae/core/data_interface.py

```diff
@@ -1174,7 +1174,12 @@ def retrieve(self, config=None, merge=True):
                 raise ValueError(
                     "To retrieve data specified in a configuration file, please input the path to your local configuration csv as a string"
                 )
-        return read_catalog_from_select(self)
+        try:
```

Comment:
this check was present in the warming levels panel, but it seemed like it belonged in the more general case